dev-requirements.txt

```diff
@@ -12,11 +12,16 @@
 # Direct dependencies:
 
 # Build distribution archive
+# Version 1.1.0 pulled after release
+build>=0.5.1, !=1.1.0
```

Comment:
Just excluding 1.1.0 does not help, as you can see in the logs. You need to pin it to <1.1.0, which is what the master branch already does.

Reply:
Agreed. Done